src/cookbook/ios-avplayer-recording.mdx

```diff
@@ -0,0 +1,43 @@
+---
+name: iOS technical note on `AVPlayer` recording
+menu: SDK Cookbooks
+route: /docs/sdk/cookbooks/ios-avplayer-recording
+showPlatformSelect: true
+---
+
+import { Title, TextBlock } from 'components'
+
+<Title>iOS technical note: AVPlayer recording</Title>
+
+<TextBlock kind="important">{`
+This guide applies to all frameworks that create or generate iOS applications.
+`}</TextBlock>
+
+Support for recording video from `AVPlayerLayer` or from camera preview via `AVCaptureVideoPreviewLayer` is not currently available.
+
+## Symptoms
+
+- When user plays video with `AVPlayer` in layer or in `AVPlayerViewController`, <b>the video is not recorded and the background (video content) is black</b>.
+
+- All camera previews are black (without content).
+
+## Technical reasons
+
+Both layers uses low level API that serves data to this layers via pixel buffers. Graphic data from `AVFoundation` are processed directly on device GPU with hardware support, not in `Core Graphics` framework like other GUI parts. For us are these low level layer inaccessible.
```

Comment:
"uses" -> "use"
"low level" -> "low-level" 
"this layers" -> "these layers"
"on device" -> "on-device"
"For us are these low level layer inaccessible." -> "For us are these low-level layers inaccessible."

Reply:
Fixed.